t/pt-archiver/archive_using_channels.t

```diff
@@ -17,67 +17,71 @@ require "$trunk/bin/pt-archiver";
 
 my $dp = new DSNParser(opts=>$dsn_opts);
 my $sb = new Sandbox(basedir => '/tmp', DSNParser => $dp);
-my $master_dbh = $sb->get_dbh_for('master');
-my $slave_dbh  = $sb->get_dbh_for('slave1'); 
+my $source_dbh = $sb->get_dbh_for('source');
+my $replica_dbh  = $sb->get_dbh_for('replica1'); 
 
-if ( !$master_dbh ) {
-   plan skip_all => 'Cannot connect to sandbox master';
+if ( !$source_dbh ) {
+   plan skip_all => 'Cannot connect to sandbox source';
 }
-elsif ( !$slave_dbh ) {
-   plan skip_all => 'Cannot connect to sandbox slave';
+elsif ( !$replica_dbh ) {
+   plan skip_all => 'Cannot connect to sandbox replica';
 } elsif ($sandbox_version lt '5.7') {
    plan skip_all => 'Only on MySQL 5.7+';
 } else {
-   plan tests => 5;
+   plan tests => 8;
 }
 
-my ($master1_dbh, $master1_dsn) = $sb->start_sandbox(
-   server => 'chan_master1',
-   type   => 'master',
+my ($source1_dbh, $source1_dsn) = $sb->start_sandbox(
+   server => 'chan_source1',
+   type   => 'source',
 );
-my ($master2_dbh, $master2_dsn) = $sb->start_sandbox(
-   server => 'chan_master2',
-   type   => 'master',
+my ($source2_dbh, $source2_dsn) = $sb->start_sandbox(
+   server => 'chan_source2',
+   type   => 'source',
 );
-my ($slave1_dbh, $slave1_dsn) = $sb->start_sandbox(
-   server => 'chan_slave1',
-   type   => 'master',
+my ($replica1_dbh, $replica1_dsn) = $sb->start_sandbox(
+   server => 'chan_replica1',
+   type   => 'source',
 );
-my $slave1_port = $sb->port_for('chan_slave1');
+my $replica1_port = $sb->port_for('chan_replica1');
 
-$sb->load_file('chan_master1', "sandbox/gtid_on.sql", undef, no_wait => 1);
-$sb->load_file('chan_master2', "sandbox/gtid_on.sql", undef, no_wait => 1);
-$sb->load_file('chan_slave1', "sandbox/slave_channels.sql", undef, no_wait => 1);
+if ( $sandbox_version lt '8.1' ) {
```

Comment:
commenting again for -lt, this time on perl, just in case. there's MySQL 9 coming, I imagine MySQL 10 would exist at a point.
```
$ perl -e "if ( '10.0' lt '8.1') { print('smaller')}"
smaller
```

Reply:
Good point! We do not care about MariaDB in our tests atm but we will in future versions. Could you please file a Jira ticket, so I can plan a fix for it?

Reply:
https://perconadev.atlassian.net/browse/PT-2390